modules/data-entry/src/main/frontend/src/questionnaire/NumberQuestion.jsx

```diff
@@ -326,7 +330,7 @@ function NumberQuestion(props) {
 
   return (
     <Question
-      defaultDisplayFormatter={isRange? rangeDisplayFormatter : undefined}
+      defaultDisplayFormatter={isRange ? rangeDisplayFormatter : unitOfMeasurement ? markdownFormatter : undefined}
```

Comment:
Why not always use the markdown formatter if it's not a range?
```suggestion
      defaultDisplayFormatter={isRange ? rangeDisplayFormatter : markdownFormatter }
```

Reply:
sure, why not